app/models/patient.rb

```diff
@@ -215,11 +216,19 @@ def search(name_or_phone_str, lines = LINES)
       all_matching_names = find_name_matches name_regexp, lines
       all_matching_phones = find_phone_matches phone_regexp, lines
 
-      (all_matching_names | all_matching_phones)
+      sort_and_limit_patient_matches(all_matching_names, all_matching_phones)
     end
 
     private
 
+
+    def sort_and_limit_patient_matches(all_matching_names, all_matching_phones)
+      matches = (all_matching_names | all_matching_phones)
+      matches.sort{|a,b|
+        b.updated_at <=> a.updated_at
```

Comment:
I love that you used the spaceship operator here, because it reminds me that the spaceship operator is a thing